src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.pkgproj

```diff
@@ -4,18 +4,16 @@
   <PropertyGroup>
     <SkipPackageFileCheck>true</SkipPackageFileCheck>
     <SkipValidatePackage>true</SkipValidatePackage>
-    <!-- replace with $(PackageVersion) once we have a new TFS build -->
-    <RuntimePackageVersion>4.3.0</RuntimePackageVersion>
   </PropertyGroup>
   <ItemGroup>
     <Dependency Include="runtime.win7-x64.runtime.native.System.Data.SqlClient.sni">
-      <Version>$(RuntimePackageVersion)-$(ExternalExpectedPrerelease)</Version>
+      <Version>$(PackageVersion)-$(SniExpectedPrerelease)</Version>
```

Comment:
Good point, this should use each package's full version directly, like [how CoreCLR picks up the full version of Microsoft.NETCore.Runtime.CoreCLR](https://github.com/dotnet/coreclr/blob/13ff59c43fcf7ddede25579a8651d7ba50ecaa31/dependencies.props#L45-L49) for example. Since the SNI runtime packages will always have a uniform version, you can point to one package id and use its version for all of them.